org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/preferences/ConsolePreferencePage.java

```diff
@@ -180,6 +181,10 @@ public void widgetSelected(SelectionEvent e) {
 
 		addField(fInterpretControlCharactersEditor);
 		addField(fInterpretCrAsControlCharacterEditor);
+
+		forceSystemEncodingEditor = new BooleanFieldEditor2(IDebugPreferenceConstants.CONSOLE_FORCE_SYSTEM_ENCODING,
```

Comment:
Should the preference be in Console or Launch Configurations.
Impacts Console but overridden in Launch Configuration so not sure..

Reply:
I've added workspace preference in case anyone wants to always go this "forced" way by default in new launch configs. If you think this is not needed, we can skip workspace preference, which simply "proposes" default value for the selection if the attribute is not set in the launch config.

Reply:
OK, I will remove it.